### ISSUE TITLE
Loading culling carousel inside a culling carousel

### ISSUE THREAD

**Issue by m0hamm3d-ac** [bbc/tal#369]
I have implementation which has a vertical carousel with large number of children horizontal culling carousel. I have issues where the children which have been culled don't load successfully. 

To work around this issue I set a minimum widget height of 5px and attach all the carousel to the main vertical carousel. Once all the horizontal carousel have finished loading I set the actual height and then realign the vertical carousel which culls the widgets not in view. However this results in performance degradation as the number of horizontal carousel grows.

The other issue faced in the similar area is the horizontal carousels(children) get detached when we navigate away from the component. When we return back to the same component the state of the detached widget is still attached in the widgetcontext. As a result of this the widgets are not visible.

Again to work around this I set the widget length to a minimum value and call the realign method. 

So to summarize - is culling carousel inside a culling carousel supported?

**Comment by basimman** [comment]
Hi m0hamm3d,
   I am also stick with the same issue, did you find any solution for this issue?  kindly share with us.

**Comment by kukulaka** [comment]
We have deprecated this project and there are no plans for active development going forward. Apologies that this issue was not replied to. 

Please see the [deprecation notice](https://github.com/bbc/tal#deprecation-announcement-12122022).
